### PR TITLE
Created a SummaryDetail component

### PR DESCRIPTION
**Pull request by miloszhygen**

### CODE REVIEW

File: src/svelte/SummaryDetail.svelte [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+  export let title: string;
+</script>
+
+
+<div class="wrapper">
```
Comment (lha047) [inline src/svelte/SummaryDetail.svelte:6]: Probably dont need this wrapper. Doesnt do anything

File: stories/components/SummaryDetail.stories.svelte [hunk 2]
```diff
@@ -1,45 +1,34 @@
...
-    disabled: {control: 'boolean'},
-    disableJs: {control: 'boolean'},
-    disableCss: {control: 'boolean'}
-  }} />
+/>
 
-<Story name="Normal" let:primary let:primaryLong let:disableCss let:disabled let:secondary>
+<Story name="Normal" let:args >
   <main class="container layout-flex-col">
     <h1>Detail summary</h1>
```
Comment (lha047) [inline stories/components/SummaryDetail.stories.svelte:15]:
```suggestion
    <h1>Oppsummering og detaljer</h1>
```
Forslag til norsk navn?

File: stories/components/SummaryDetail.stories.svelte [hunk 3]
```diff
@@ -1,45 +1,34 @@
...
-          <ul>
-            <li>Hvor skjedde det?</li>
-            <li>Her er andre punkt i listen</li>
-          </ul>
-          <p>Så er det noe tekst på bunnen av summary</p>
-        </div>
-      </details>
-    </div>
+    <SummaryDetail title={args.title}>
+      <p>
```
Comment (lha047) [inline stories/components/SummaryDetail.stories.svelte:17]: Would be nice if the details also could be edited in Storybook action control

File: stories/components/SummaryDetail.stories.svelte [hunk 4]
```diff
@@ -1,45 +1,34 @@
 <script lang="ts">
-  import {Meta, Story} from '@storybook/addon-svelte-csf'
-  import FileUploadButton from '../../src/svelte/FileUploadButton.svelte'
+  import {Meta, Story} from '@storybook/addon-svelte-csf';
+  import SummaryDetail from '../../src/svelte/SummaryDetail.svelte';
 </script>
 
 <Meta
   title="Components/Summary Detail"
```
Comment (lha047) [inline stories/components/SummaryDetail.stories.svelte:7]: Klarer vi å finne et norsk navn på denne? :P Oppsummering og detaljer?

File: src/svelte/SummaryDetail.svelte [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  <details>
+    <summary>{title}</summary>
+    <div class="summary-wrapper">
+      <slot />
+    </div>
+  </details>
+</div>
```
Comment (lha047) [inline src/svelte/SummaryDetail.svelte:13]: Add an empty line

**Review by lha047 — CHANGES_REQUESTED** [review]
Nice work :D

Just have some small comments

File: stories/components/SummaryDetail.stories.svelte (hunk 4 above)
Comment (miloszhygen) [reply stories/components/SummaryDetail.stories.svelte:7]: If I change `title="Components/Summary Detail"` to `title="Components/Oppsummering og Detaljer"` I get an error "Are you sure a story with that id exists?" Maybe I have to change the component file name too?

File: src/scss/components/_legal-collection.scss [hunk 6]
```diff
@@ -0,0 +1,8 @@
+@use '../theme' as *;
+
+.legal-collection {
+  &__border-top{
+    border-top: $border-width-regular solid var(--color-mt-beige);
+    margin: var(--spacer-medium) 0;
+  }
```
Comment (miloszhygen) [inline src/scss/components/_legal-collection.scss:7]: @lha047 I added margin top and bottom to this class

**Review by lha047 — APPROVED** [review]
Nice work :D

Would like for it to have tests as well, but closing and merging for now :D

**Pull request merged**